instrumentation/kafka-streams/pom.xml

```diff
@@ -64,6 +64,14 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <!-- used to test checked exceptions from transformers -->
```

Comment:
Can consider defining it yourself instead of adding lombok which is surprising to see in a project not using its codege , just a few lines.

https://github.com/line/armeria/blob/master/core/src/main/java/com/linecorp/armeria/common/util/Exceptions.java#L197